to-do-list/src/components/List.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Todo from './Todo'
+
+// temporary static data
+const tasks = [
+  {id:1, task: "Dance a lot"},
+  {id:2, task: "Run a mile"},
+  {id:3, task: "Drink some soda"},
+  {id:4, task: "Task #4"},
+  {id:5, task: "Computers"},
+  {id:6, task: "Code"}
+]
+
+const TodoList = (props) => {
+  const todos = props.tasks
+
+  return (
+    <tbody>
+      { todos.map((todo) =>
+        <Todo value={todo.task} />
```

Comment:
does Todo need a key passed to it since there are multiple node elements
```
{ todos.map((todo) =>
   <Todo key={todo.id} value={todo.task} />
)}
```